package.json

```diff
@@ -89,7 +89,10 @@
     "url": "^0.11.0",
     "webdriver-manager": "^12.1.0",
     "x-frame-options": "^1.0.0",
-    "zone.js": "^0.8.26"
+    "zone.js": "^0.8.26",
+    "@hmcts/document-viewer-webcomponent": "^1.1.5",
```

Comment:
are these dependencies actually needed (directly)?

Reply:
Safely removed after analysing the impacts.